org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -65,9 +65,28 @@ define([
             formData: [{
               entity_table: customSettings.entityTable || error('entityTable'),
               crm_attachment_token: customSettings.crmAttachmentToken || error('crmAttachmentToken')
+            }],
+            filters: [{
+              name: 'fileFormatFilter',
+              fn: function (item /*{File|FileLikeObject}*/ , options) {
+                var type = '|' + item.type.slice(item.type.lastIndexOf('/') + 1) + '|';
```

Comment:
indexOf alone will not give correct result because sometimes a mimetype may be part of another type (for eg. x-bzip and x-bzip2) and may not give correct index. 

The includes logic also will fail for above condition.